module/tello/test/demo/main.c

```diff
@@ -0,0 +1,190 @@
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <wallaby/wpa_ctrl.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <unistd.h>
+
+#include <wallaby/camera.h>
```

Comment:
Are these supposed to be wallaby or kipr?

Reply:
sorry for the delayed response, I was out of town. These fixes should answer the mail